src/index.ts

```diff
@@ -305,7 +305,8 @@ You may find the requirements here: ${cyan('https://tauri.app/v1/guides/getting-
       answers: recipeAnswers ?? {},
     });
   }
-
+  const validName =  isValidPackageName(appName) ? appName : toValidPackageName(appName) || defaults.appName
+  cfg.appName = validName
```

Comment:
instead of this, just use `validName` instead of `cfg.appName` here https://github.com/tauri-apps/create-tauri-app/blob/e17dc70c319b0558f8be98e738f30ad76a5d2152/src/index.ts#L341